astropy/stats/sigma_clipping.py

```diff
@@ -950,10 +1278,9 @@ def sigma_clipped_stats(
         removed. The default is ``'std'``.
 
     std_ddof : int, optional
-        The delta degrees of freedom for the standard deviation
-        calculation. The divisor used in the calculation is ``N -
-        std_ddof``, where ``N`` represents the number of elements. The
-        default is 0.
+        The delta degrees of freedom for the standard deviation. The
+        divisor used in the calculation is ``N - std_ddof``, where ``N``
+        represents the number of elements. The default is 0.
```

Comment:
Perhaps not really for this PR, but should one note here that if the mean is not known a priori, the correct value is 1?

Reply:
I'll open a separate PR for that.  I think the same description will need to be updated in a few places.

Reply:
I've updated the descriptions in this PR and https://github.com/astropy/astropy/pull/17238 for the other one.